workload/master_config.yaml

```diff
@@ -0,0 +1,43 @@
+s3api:
+  # TEST_ID: TEST-35748
+  object_size:
+    start: 0Kib
+    end: 100Kib
+  part_range: #Multipart Specific
+    start: 50
+    end: 100
+  min_runtime_to_pass: 2h
+  sessions_per_node: 1
+  range_read: 100bytes
+  tool: s3api
+  operation: bucket/object_range_read/object_fix_size/object_random_size/copy_object/multipart/multipart_partcopy
+  wait_on_operation: True/False # True: Wait till operation completes to mark pass,
+  #if False , test will be marked pass as per minimum run time
+
+mix_io:
+  # TEST_ID: TEST-40042
+  object_size: #(Single input, List, and Dictionary{key:value})  object size =
+  # for 10000 samples(user configurable) and distribution input may be like {1kb :10%, 100kb: 20%, 1000kb: 30%, 10000kb: 40%}
+    start: 0bytes
+    end: 1Gib
+  write_percentage: 100  # Percentage of data to fill the storage.
+  read_percentage: 100  # Percentage of data to be used to read from storage.
+  delete_percentage: 0  # Percentage of data to be deleted from storage.
+  cleanup_percentage: 0  # Percentage of data once reached then cleanup all data.
+  #  If None then it will be fetched from cluster(cortx specific.)
+  total_storage_size: None  # Total available storage from cluster or user to be used.
+  min_runtime_to_pass: 30d  # Minimum execution durations to mark workload pass.
+  sessions_per_node: 20  # Number of sessions per node.
+  background_delete: False  # Enable/Disable background delete(Cortx specific).
+  sample_size: 10000 # int (any user defined numbers) # samples should be user configurable like 10000
+  tool: s3api/s3bench
+  operation: "mix_object_ops"
+
+degraded_io:
+  pod_downtime: 20s #time taken to degraded pods (IO sleep/halt time), shutdown time #i.e. 20s
+  pod_downtime_schedule: 2h #time when nodes/pods need to be degraded [Time Delta in (HH:MM:SS)], before this IO will be in happy path
+  schedule_frequency: 1 #once or repetitive (after every schedule time interval) --> repetition will happen as per schedule time interval
+  pod_uptime_schedule: 2  #time when pods need to be active back since downtime. # when pods need to be reverted online[Time Delta in (HH:MM:SS) ]
```

Comment:
Pod uptime schedule should also accept none to support permanent failure.

Reply:
Ok, Mentioned and will have support for same.